ms2query/utils.py

```diff
@@ -105,6 +103,44 @@ def get_classifier_from_csv_file(classifier_file_name: str,
     else:
         results = pd.concat(list_of_classifiers, axis=0, ignore_index=True)
 
-    results["inchi_key"] = list_of_inchikeys
-    results.rename(columns={"inchi_key": "inchikey"}, inplace=True)
+    results["inchikey"] = list_of_inchikeys
     return results
+
+
+def columns_and_order_for_output_dataframe(return_standard_columns: bool,
+                                           return_classifier_columns: bool,
+                                           additional_metadata_columns: List[str] = None,
+                                           additional_ms2query_score_columns: List[str] = None) -> List[str]:
+    """Returns the column names in order for the output of results table
+
+    This is used by the functions MS2Library.analog_search_store_in_csv, ResultsTable.export_to_dataframe
+    and get_classifier_from_csv_file
+
+    Args:
+    ------
+    return_standard_columns:
+        If true the standard columns are returned,
+        if additional_metadata_columns or additional_ms2query_score_columns is specified these are appended.
+        If return_classifier_columns is True, the classifier_columns are also appended to the columns list.
+    return_classifier_columns:
```

Comment:
The parameters names confused me a bit.
When a function ask for a bool names `return_standard_columns` I would assume that this will get me a default set/list.
I guess one issue here is that it is not so easy to define what classifier columns are ("smiles" for instance could also end up in the `additional_metadata_columns` as well)

So, one question I have: Do you ever need to only get the "classifier column names"?
If not, then I would only have options such as `additional_columns=List`  and `add_classifier_columns=True/False`.
(and also check for duplicates)

Reply:
Hmm yes, I can see the confusion. 

The name standard_columns is a bit confusion. Best alternative I could come up with is return_non_classifier_columns

This function just adds the additional_metadata_columns and the additionatial_ms2query_score_columns. For this function there is no distinction between these functions. Still it is best to add them together here, so the order is always the same and no accidental mistakes are made were these columns are added together in different ways in different functions. Besides this I cannot just add these columns together before passing to this function, since if they are None, it will raise an error for appending a list to None. 

For the function get_classifier_from_csv_file() you want to only get the classifier_columns. So column_names_for_output(return_standard_columns=False, return_classifier_columns=True).

Reply:
To further clarify. For this function the distinction between the different kind of columns does not really matter. It is mostly meant for having one central place where the default columns are set, since before this was done 3 times in 3 different functions, which was very sensitive to introducing errors in the output data (non matching headers and values). If I now want to change the order or information returned, I can just change it here without needing to double check if I did change it in all the needed functions. Do the current changes made clarify this a bit?